clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -23,7 +23,7 @@ let wsPath: string;
 let filePath: string;
 beforeAll(() => {
   (vscode.workspace.workspaceFolders as any) = [
-    { uri: { fsPath, path: fsPath } } as any,
+    { uri: { fsPath: makePath(fsPath), path: makePath(fsPath) } } as any,
```

Comment:
The `path:` will need a special handling - it is a URI path, not true FS path.
`path: (process.platform == "win32" ? "/a:" : "") + fsPath`

Reply:
should resolve : bd7407aff3fd51fb7a75ded9f2168aac8ae0dc86